app/models/case_log.rb

```diff
@@ -431,6 +432,15 @@ def get_totadult
     end
   end
 
+  def get_refused
+    return 1 if [age1_known, age2_known, age3_known, age4_known, age5_known, age6_known, age7_known, age8_known].any?(1)
+    return 1 if [sex1, sex2, sex3, sex4, sex5, sex6, sex7, sex8].any?("R")
+    return 1 if [relat2, relat3, relat4, relat5, relat6, relat7, relat8].any?(3)
```

Comment:
Same here, from the ticket seems like it's expecting `R` instead of 3. Are we saving it in the db differently?

Reply:
Yes - `3` is `prefers not to say`

Reply:
Does that match what's in the data dictionary?